CHANGES

```diff
@@ -1,3 +1,53 @@
+1.7.1 -> 1.7.2
+	- NNStreamer for Edge-AI project started.
+		- Main festures of 1.8.0 release and its immediate successors will be "Edge-AI", which allows distributed on-device AI inferences.
+
+		- The new stream type, "Flex-Tensor", is introduced. Dimensions and types of tensor stream may vary per frame without cap-renegotiations.
+			- Many nnstreamer's tensor-* elements support Flex-Tensor.
+			- You may use tensor-converter to convert between flex-tensor and (static) tensor.
+		- MQTT-SINK and MQTT-SRC elements are added for edge-AI systems with MQTT pub/sub streams.
+			- MQTT streams support "ANY" capabilities.
+			- Assuming that clocks of nodes are synchronized by NTP or other mechanisms, pipeline users may send timestamp related info via MQTT streams for multi-source synchronization.
+		- Tensor-crop, a new nnstreamer-gstreamer element.
+			- Basic feature only (cropping a tensor stream with information of another tensor stream)
+
+	- Major features
+		- GSTPipeline to PBTXT parser. You can use PBTXT-pipeline visualization tools with the parsed results.
+		- TVM support
+
```

Comment:
Is this an intended blank line?

Reply:
Yes. that's between "major of major" and "minor of major" .